qsv install failed

When I install the qsv with ./gradlew :qsv:build,I got the error message like thses :org.qcmg.qsv.util.QSVUtilTest > testGetAnalysisId FAILED
    java.lang.AssertionError at QSVUtilTest.java:172

324 tests completed, 1 failed
:qsv:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':qsv:test'.

So how can I solve it? thanks!

--- a/qsv/test/org/qcmg/qsv/util/QSVUtilTest.java
+++ b/qsv/test/org/qcmg/qsv/util/QSVUtilTest.java
@@ -7,6 +7,7 @@ import static org.junit.Assert.assertTrue;
 import java.io.File;
 import java.io.IOException;
 import java.util.Date;
+import java.util.Locale;
 
 import org.junit.Rule;
 import org.junit.Test;
@@ -168,9 +169,11 @@ public class QSVUtilTest {
 	}
 
 	@Test 
-	public void testGetAnalysisId() {    	
-		assertEquals("qSV_test_20121511_1544", QSVUtil.getAnalysisId(false, "test", new Date(1352958269803L)));
-//		assertTrue(QSVUtil.getAnalysisId(false, "test", new Date(1352958269803L)).contains("qSV_test_20121511_1544"));
+	public void testGetAnalysisId() {
+		/*
+		 * Due to timezone differences, it is safest just to dictate which year we expect the analysis id to start with
+		 */
+		assertEquals(true, QSVUtil.getAnalysisId(false, "test", new Date(1352958269803L)).startsWith("qSV_test_2012"));
 		assertTrue(QSVUtil.getAnalysisId(true, "test", new Date(1352958269803L)).length() == 36);
 	}
 

